.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": false,
+  "printWidth": 80
```

Comment:
Not needed as `80` is the default value

I'd recommend to just not add a prettierrc file. Seems unnecessary.